charts/vc-authn-oidc/README.md

```diff
@@ -140,7 +138,7 @@ kubectl delete secret,pvc --selector "app.kubernetes.io/instance"=my-release
 | Name                                                  | Description                                                                                                                          | Value                                         |
 | ----------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------ | --------------------------------------------- |
 | `acapy.enabled`                                       | Deploy AcaPy agent instance                                                                                                          | `true`                                        |
-| `acapy.agentUrl`                                      | Agent host, required if `enabled`` is `false`, otherwise ignored                                                                     | `""`                                          |
+| `acapy.agentUrl`                                      | Agent host, required if ` enabled`` is  `false`, otherwise ignored                                                                   | `""`                                          |
```

Comment:
This change still doesn't fix the code highlighting:
![image](https://github.com/bcgov/vc-authn-oidc/assets/2119240/cca035b1-a34b-4277-a79e-b9a4bfc4ef7a)